src/topology/algebra/ordered/basic.lean

```diff
@@ -2267,12 +2267,12 @@ lemma is_glb.exists_seq_monotone_tendsto [first_countable_topology α]
                         tendsto u at_top (𝓝 x) ∧ (∀ n, u n ∈ t) :=
 htx.exists_seq_monotone_tendsto' ht (is_countably_generated_nhds x)
 
-lemma exists_seq_strict_antimono_tendsto' [densely_ordered α]
+lemma exists_seq_strict_antitone_tendsto' [densely_ordered α]
```

Comment:
can you fix the name (`strict_antitone` should be `strict_anti`) and also use the new terminology in the statement? Same thing for the next lemma.